sources/editor/Stride.Core.Assets.Editor/ViewModel/AssetViewModel.cs

```diff
@@ -715,5 +683,33 @@ public void PrepareSave(ILogger logger)
         {
             PropertyGraph?.PrepareForSave(logger, assetItem);
         }
+
+        public void UpdateAsset(Asset newAsset, ILogger loggerResult)
```

Comment:
Any particular reason to move the method here? Usually we have public method first, then protected, then private.

Reply:
Moved it back, I'll let @xen2 chime in if this was on purpose